frontend/packages/integration-tests-cypress/reporter-config.json

```diff
@@ -1,7 +1,14 @@
 {
-  "reporterEnabled": "spec, mocha-junit-reporter",
+  "reporterEnabled": "mocha-junit-reporter, mochawesome",
   "mochaJunitReporterReporterOptions": {
     "mochaFile": "../../gui_test_screenshots/junit_cypress-[hash].xml",
     "toConsole": false
+  },
+  "mochawesomeReporterOptions": {
+    "reportDir": "../../gui_test_screenshots/",
+    "reportFilename": "junit_cypress",
+    "overwrite": false,
```

Comment:
Does this mean the report will fail if it's already there? I don't see this in the options, but maybe it's not documented.

https://github.com/adamgruber/mochawesome#available-options

Reply:
hmm...I copied this from https://docs.cypress.io/guides/tooling/reporters.html#Config-file-4, but yeah, they don't match up?
Oh, I see, line under 'Avail Options' reads `The options below are specific to the reporter. For a list of all available options see mochawesome-report-generator options.`
We do need overwrite:
https://github.com/adamgruber/mochawesome-report-generator#overwrite